www/class/centreonConfigCentreonBroker.php

```diff
@@ -684,38 +684,86 @@ public function insertConfig($values)
     {
         $objMain = new CentreonMainCfg();
```

Comment:
I think the function is too big and it does more than one thing : creates records into "cfg_centreonbroker" table and into "cfg_centreonbroker_log" table.
Sometimes it has no returns and thrown exceptions aren't handled (maybe they should be logged ?)
I think we should split it in 3 parts and make them more simple.